.github/workflows/build.yml

```diff
@@ -121,17 +128,22 @@ jobs:
           echo "    flags: +openssl"      >> cabal.project.local
 
       - name: Unix build CLI
-        id: unix_cli_build
+        id: unix_build
```

Comment:
why rename?

Reply:
for consistency across the build condition here.
nvm, the split wasn't needed.